packages/manager/apps/pci-public-ip/src/pages/onboarding/OnBoardingPage.tsx

```diff
@@ -0,0 +1,200 @@
+      },
+      href: GUIDES.configure_floating_ip[ovhSubsidiary],
+    },
+  ];
+
+  return (
+    <>
+      {project && <OsdsBreadcrumb items={breadcrumbItems} />}
+      {isDiscoveryProject(project) && (
+        <div className="mb-8">
+          <PciDiscoveryBanner projectId={projectId} />
+        </div>
+      )}
+      <OnboardingLayout
+        title={t('pci_additional_ips_title')}
+        description={
+          <>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+            >
+              {tOnBoarding('pci_additional_ips_onboarding_content1')}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              className="mt-4 block"
+            >
+              {tOnBoarding('pci_additional_ips_onboarding_content2')}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              className="mt-6 block"
+            >
+              {tOnBoarding('pci_additional_ips_onboarding_content3')}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+              className="block mt-6"
+            >
+              {tOnBoarding('pci_additional_ips_onboarding_content4_heading')}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              className="block"
+            >
+              {tOnBoarding(
+                'pci_additional_ips_onboarding_content4_description',
+              )}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+              className="block mt-6"
+            >
+              {tOnBoarding('pci_additional_ips_onboarding_content5_heading')}
+            </OsdsText>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              className="block"
+            >
+              {tOnBoarding(
+                'pci_additional_ips_onboarding_content5_description',
+              )}
+            </OsdsText>
+          </>
+        }
+        orderButtonLabel={tOnBoarding(
+          'pci_additional_ips_onboarding_action_buy',
+        )}
+        onOrderButtonClick={() => {
+          if (!isDiscoveryProject(project)) navigate('./new');
+        }}
```

Comment:
Instead i would have preferred to create a callback and move this code inside for better readability